.github/workflows/test.yml

```
@@ -15,11 +15,22 @@ jobs:
+
+  test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        params:
+          - name: 'Params: image'
+            image: ghcr.io/joschi/dive:latest
+            github-token: ''
+          - name: 'Params: image, github-token'
+            image: ghcr.io/joschi/dive:latest
+            github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.4)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)